source/2019-12-20-octane-is-here.md

```diff
@@ -0,0 +1,340 @@
+## Getting Started
+
+Whether you're a new Ember developer, coming back to Ember after many years, or an existing Ember developer, the fastest and easiest way to get learn how to build applications the Octane way is to run through [the refreshed tutorial](https://guides.emberjs.com/release/tutorial/part-1/).
+
+Once you got through the tutorial, it's time to build something real for fun. The Ember addon ecosystem is a great part of Ember, so you'll want to reach for addons to speed up the process of building your project.
+
+[Ember Observer](https://emberobserver.com/) is a directory for the Ember addon ecosystem. Each addon gets a quality score based on a human review of formal criteria like the existence of a meaningful README, whether the addon has an automated build, and whether the addon is maintained by more than one person. This week, it will also indicate whether an addon is Octane Ready.
+
+Because of the details of Octane's compatibility story, most addons should be Octane Ready without any changes. Ember Observer will help the community proactively identify and fix Octane problems in maintained packages.
+
+## A Closer Look at Seamless Interop
+
+<!--alex ignore special-->
+In addition to the elimination of computed properties, the Glimmer reactivity model also doesn't include special Ember proxies or observers. The Octane reactivity model is more powerful than the classic one, but it's much easier to use.
+
+The Octane reactivity model wouldn't be very useful for existing Ember users if it was hard to use objects implemented using the classic reactivity model from objects implemented using the Octane model. For that reason, we worked hard to ensure that existing Ember applications can freely use classic objects in classes built using tracked properties.
+
+```js
+class Contact {
+  @tracked person;
+
+  constructor(person) {
+    this.person = person;
+  }
+
+  get description() {
+    return this.person.description;
+  }
+}
+
+import EmberObject, { computed } from '@ember/object';
+
+const Person = EmberObject.extend({
+  firstName: null,
+  lastName: null,
+  age: null,
+  country: null,
+
+  fullName: computed('firstName', 'lastName', function() {
+    return `${this.firstName} ${this.lastName}`;
+  }),
+
+  description: computed('fullName', 'age', 'country', function() {
+    return `${this.fullName}; Age: ${this.age}; Country: ${this.country}`;
+  })
+});
+
+let captainAmerica = new Person({
+  firstName: 'Steve',
+  lastName: 'Rogers',
+  age: 80,
+  country: 'USA'
+});
+
+let contact = new Contact(captainAmerica);
+contact.description; // "Steve Rogers; Age: 80; Country: USA"
+captainAmerica.set('firstName', 'Steven');
+contact.description; // "Steven Rogers; Age: 80; Country: USA"
+```
+
+Because these two systems are interoperable, libraries can adopt the Octane reactivity system without a breaking change to their API.
+
+This work also allows existing Ember codebases to adopt Octane patterns module-by-module.
+
+## Octane:
```

Comment:
```suggestion
## Summary

We believe that developers should have fun while building for the web, and we think the Octane edition of Ember makes it possible to have both fun and stability in our work. 

We also could not have done this without the effort of the community and every member of the [Ember Core Teams](https://emberjs.com/team/). Thank you for believing in this project, contributing to this project, and continuing to help Ember be the best it can be. 
```